src/test/scala/ch.zzeekk.spark.temporalquery/UDFTest.scala

```diff
@@ -0,0 +1,48 @@
+package ch.zzeekk.spark.temporalquery
+
+import java.sql.Timestamp
+
+import TemporalQueryUtil._
+import TestUtils._
+import UDF._
+
+class UDFTest extends org.scalatest.FunSuite {
```

Comment:
```suggestion
class UDFTest extends FunSuite {
```
And import org.scalatest.FunSuite

Reply:
Done: https://github.com/zzeekk/spark-temporalquery/pull/11/commits/26ee7ae49877695da4343f997eff239aa7e73989